src/Application/Actions/Donations/Update.php

```diff
@@ -142,11 +142,37 @@ protected function action(Request $request, Response $response, array $args): Re
                     $retryCount,
                 ));
 
-                \usleep(100_000 * (2 ** $retryCount)); // pause for 0.1, 0.2, 0.4 and then 0.8s before giving up.
+                $microseconds = (int)(100_000 * (2 ** $retryCount)); // pause for 0.1, 0.2, 0.4 and then 0.8s before giving up.
+                \assert($microseconds >= 0);
+                \usleep($microseconds);
```

Comment:
I think Psalm isn't clever enough to know that $microseconds is always positive without us asserting it, there was something in the baseline but because of a mistake I made psalm stopped looking at that and I removed it from the baseline. May as well leave this with the assert now.